test/quadratic.jl

```diff
@@ -63,6 +63,11 @@ for x in 2:xmax-1
     @test_approx_eq A[x] itp3[x]
 end
 
+# Rational element types
+A = Rational{Int}[x^2//10 for x in 1:10]
+itp = Interpolation(A, Quadratic(Free(),OnCell()), ExtrapNaN())
+@test itp[11//10] == (11//10)^2//10
```

Comment:
hmm, it's very strange. On julia 0.4:
```
julia> itp.coefs
12-element Array{Rational{Int64},1}:
  -1//40                                                                                                                                                                                    
   3//40                                                                                                                                                                                    
   3//8                                                                                                                                                                                     
   7//8                                                                                                                                                                                     
  63//40
  99//40
 143//40
  39//8 
  51//8 
 323//40
 399//40
 483//40
```
On julia 0.3:
```
julia> itp.coefs
12-element Array{Rational{Int64},1}:
  6899716117846699271//8619614680507807464
  7936969560604716305//5916187127003196984
  2250090984434330863//6323126675786419496
 -4372087249599443041//7183461070236085576
 -7973312573468218199//2373339102190889880
  9203035543489481799//6118447631419504648
   336340586073110987//6949707296401608   
  4469594703982377491//8584938456800293288
 -3056145513736821459//2910278932930701560
  -599620387571391593//1277212073810152584
 -5424053943932394579//786135081844975736 
 -8232532661296565515//432728054263014168 
```
My strong suspicion is that something changed (for the better) in the linear algebra code in julia 0.4, probably specifying constants as Rationals in whatever solver gets called in the prefiltering step.

Also on 0.4, any conversion that causes an overflow will result in an error, whereas on 0.3 it will silently (and incorrectly) succeed.